tests/integration/local/request_multiple_nodes_test.py

```diff
@@ -13,34 +12,32 @@
 
 
 @pytest.fixture(scope="function")
-def node_1():
-    name = secrets.token_hex(4)
-    print(name)
-    node = sy.Orchestra.launch(
-        name=name,
-        dev_mode=True,
+def node_1(faker):
+    node = sy.orchestra.launch(
```

Comment:
dont trust faker.name(), it wont be unique. I will add some changes that fix it (less pretty name, but whatever)